src/docs/asciidoc/web/webflux.adoc

```diff
@@ -2266,9 +2266,20 @@ configure or customize message readers.
 You can use `@RequestBody` in combination with `javax.validation.Valid` or Spring's
 `@Validated` annotation, which causes Standard Bean Validation to be applied.
 By default, validation errors cause a `WebExchangeBindException`, which is turned
-into a 400 (`BAD_REQUEST`) response. Alternatively, you can handle validation errors locally
-within the controller through an `Errors` or a `BindingResult` argument. The following
-example uses a `BindingResult` argument`:
+into a 400 (`BAD_REQUEST`) response. 
+
+[source,java,indent=0]
+[subs="verbatim,quotes"]
+----
+@ExceptionHandler(WebExchangeBindException.class)
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+@ResponseBody
+public Mono<T> processValidationError(WebExchangeBindException ex) {
+	//..
+}
+----
```

Comment:
It's unclear what you mean with this update. The paragraph above talks about the built-in, exception handling that happens by default and does not require the sample code shown, which shows how to handle the exception explicitly to the same effect..